lib/const.rb

```diff
@@ -1,6 +1,6 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.8.11'
+CREW_VERSION = '1.8.11.1'
```

Comment:
It's fine to go ahead and increment to `1.8.12`.  I don't want us to get into crazy versioning.

Reply:
Sure. :)